composer.json

```diff
@@ -10,18 +10,20 @@
     ],
     "require": {
         "php": "^5.6 || ^7.0",
-        "php-http/httplug": "^1.0",
+        "php-http/httplug": "^1.0 || ^2.0",
         "php-http/client-implementation": "^1.0",
-        "php-http/discovery": "^1.0",
         "psr/log": "~1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8.36 || ^5.5 || ^6.2",
+        "phpunit/phpunit": "^4.8.36 || ^5.5 || ^7.5 || ^8.4",
```

Comment:
Hm, I'd say we need to keep `^6.2` as well, otherwise we will force devs who have PHPUnit 6 to install a newer major PHPUnit without any reasons